com.unity.mobile.notifications/Editor/NotificationSettingsProvider.cs

```diff
@@ -20,9 +20,9 @@ internal class NotificationSettingsProvider : SettingsProvider
 
         private readonly string[] k_ToolbarStrings = {"Android", "iOS"};
         private const string k_InfoStringAndroid =
-            "Only icons added to this list or manually added to the `res/drawable` folder can be used by notifications.\n\n" +
-            "Small icons can only be composed simply of white pixels on a transparent backdrop and must be at least 48x48 pixels.\n" +
-            "Large icons can contain any colors but must be not smaller than 192x192 pixels.";
+            "Only icons added to this list or manually added to the 'res/drawable' folder can be used for notifications.\n\n" +
+            "Small icons must be 48x48px at least and composed of white pixels on a transparent background.\n" +
```

Comment:
"Small icons must be at least 48x48px and composed...

Reply:
Fixed.